src/main/java/core/basesyntax/model/state/FigureSupplier.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.model.state;
+
+import core.basesyntax.Figure;
+import core.basesyntax.model.figures.Circle;
+import core.basesyntax.model.figures.IsoscelesTrapezoid;
+import core.basesyntax.model.figures.Rectangle;
+import core.basesyntax.model.figures.RightTriangle;
+import core.basesyntax.model.figures.Square;
+
+/**
+ * Класс должен возвращать случайный объект Figure.
+ * Создать массив существующих фигур.
+ * Создать метод, который вернет случайный объект Figure.
+ */
+
+public class FigureSupplier {
+
+    private static final Figure[] FIGURES = new Figure[]{
+            new Circle(), new Rectangle(), new Square(),
+            new RightTriangle(), new IsoscelesTrapezoid()
+    };
```

Comment:
You need to generate figures with random params

Reply:
getFigure() returns an object with random values!

Reply:
Return figure object with random values in `generateFigure()` method

Reply:
okey, i get it.